packages/integrations/netlify/README.md

```diff
@@ -72,6 +72,56 @@ export default defineConfig({
 });
 ```
 
+### Run middleware in Edge Functions
+
+When deploying to Netlify Functions, you can choose to use an Edge Function to run your Astro middleware.
+
+To enable this, set the `build.excludeMiddleware` Astro config option to `true`:
+
+```js ins={9}
```

Comment:
```suggestion
```js
```

Is it needed?

Reply:
It will be picked up in the docs site to highlight that line as an “insertion”! Doesn’t do anything in regular Markdown on GitHub/npm.